src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -268,20 +268,13 @@ internal static class ChannelHelper
         public static async Task<RestUserMessage> SendMessageAsync(IMessageChannel channel, BaseDiscordClient client,
             string text, bool isTTS, Embed embed, AllowedMentions allowedMentions, MessageReference messageReference, MessageComponent component, ISticker[] stickers, RequestOptions options, Embed[] embeds)
         {
+            embeds ??= Array.Empty<Embed>();
             if (embed != null)
-            {
-                if (embeds == null)
-                    embeds = new[] { embed };
-                else
-                {
-                    List<Embed> listEmbeds = embeds.ToList();
-                    listEmbeds.Insert(0, embed);
-                }
-            }
+                embeds = new[] { embed }.Concat(embeds).ToArray();
```

Comment:
Can you implement the `.Prepend` function?

Reply:
I don't know how to implement it. Isn't it just simpler to do it this way? I mean, `Prepend` will just create a new IEnumerable that has an item prepended, and creating a new array with the item + `Concat` is functionally the same.

Reply:
Fair enough